cfme/tests/infrastructure/test_vmware_provider.py

```diff
@@ -249,7 +254,31 @@ def test_vmware_cdrom_dropdown_not_blank():
             4.Virtual machine is selected
             5.Dropdown of ISO files is not empty for CD/DVD Drive
     """
-    pass
+    datastore_collection = appliance.collections.datastores
+    ds = [ds.name for ds in provider.data['datastores'] if ds.type == 'iso']
+    try:
+        iso_ds = datastore_collection.instantiate(name=ds[0], provider=provider)
+    except IndexError:
+        pytest.skip('No datastores found of type iso on provider {}'.format(provider.name))
+    iso_ds.run_smartstate_analysis()
+    vms_collections = appliance.collections.infra_vms
+    vm = vms_collections.instantiate(name='cu-24x7', provider=provider)
+    view = navigate_to(vm, 'Reconfigure')
+    actions_column = view.cd_dvd_table[0]['Actions']
+    # First disconnect if already connected
+    if actions_column.text == 'Connect Disconnect':
```

Comment:
Doing some local testing in 5.10, looks like it is "Disconnect" and not "Connect Disconnect"

Reply:
If you read the text on `action_column.text` it does actually read text from both buttons. Give it a try. I did use only `Disconnect` first and didn't work.

Reply:
![image](https://user-images.githubusercontent.com/8180469/59876672-38cca000-9372-11e9-8877-2a996b935519.png)

Reply:
As you can see that UI shows only `Disconnect` but `.text ` returns both.

Reply:
I think this may be a UI bug